src/services/refactors/moveToNewFile.ts

```diff
@@ -954,6 +978,22 @@ function isExported(sourceFile: SourceFile, decl: TopLevelDeclarationStatement,
         getNamesToExportInCommonJS(decl).some(name => sourceFile.symbol.exports!.has(escapeLeadingUnderscores(name)));
 }
 
+function getDefaultExportName(sourceFile: SourceFile) {
+    return tryCast((sourceFile.symbol.exports?.get("default" as __String)?.declarations?.[0] as undefined | ExportAssignment)?.expression, isIdentifier);
+}
+
+function isExportedLaterInOldFile(sourceFile: SourceFile, decl: TopLevelDeclarationStatement, useEs6Exports: boolean, name: Identifier, toMove: readonly Statement[]): boolean {

```

Comment:
Could you explain what this function is trying to do, and why we need each of the conditions in the `if` below?

Reply:
You can think of it as modified version of `isExported` function that is declared above. The last check is used to figure out wether the export itself is included into symbols that are going to be moved (`toMove`), if not, symbol is added into `symbolsExportedLaterInOldFile` map.